libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-pie-chart/eo-pie-chart.component.ts

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
issue: A chart is usually a molecule, not an atom.

Reply:
Moved the chart into the energy of origin lib, so it is no longer part of atomic-design